Hugo.gitignore

```diff
@@ -0,0 +1 @@
+/public/
```

Comment:
I suspect the more correct would be "public/"

Reply:
leading with a forward slash is the better approach. It ignores only the one directory called public in the root of the git repository. Without a leading slash, 'public/' ignores all directories that are called public, no matter where they are.